### DISCUSSION TITLE
Public and private pages in the same documentation site?

### DISCUSSION THREAD

**Discussion by cairo-me** [markdoc/markdoc#47] [Q&A] [answered]
First of all, terrific work here by the Stripe team. What a beautiful documentation platform!

I'm currently looking for a good documentation platform for our company, and am curious if it's possible to have both public and internal facing documentation on the same documentation site with Markdoc. In our environment, we would like all company documentation to be in the same place and then mark a doc page as internal or public based on the nature of the content.

Is this currently possible or something that is on the potential roadmap? Presumably, this would also mean some sort of authentication/login mechanism, too...

**Comment by mfix-stripe** [comment, marked as answer]
Hey @cairo-me, yes this is totally possible, and is something we do constantly on https://stripe.com/docs. 
Markdoc itself is a low level library, so it isn't prescriptive about how you should handing this gating mechanism. 

You can also use conditional tags (https://markdoc.io/docs/tags#if/else) to control what content shows up within a page.

**Reply by jeshu2002** [reply]
I want to use an SSO login. How is it possible?

**Comment by jeshu2002** [comment]
Yes, I am also facing the same problem with Markdoc, I am unable to integrate with Auth0.
